docs/how-to-guides.md

```diff
@@ -211,3 +211,20 @@ your virtualenv active), run:
 Then you can run the tests:
 
     py.test
+
+
+## How to use custom events
+
+Roll as a very small API for listening and sending event. It's possible to use
+it in your project for your own events.
+
+That would look like this:
+
+    app = Roll()
+
+    @app.listen('myevent')
+    def listener(myarg):
```

Comment:
What would be a real use-case? Sending emails? Filling metrics?

Reply:
Added a more realistic use case. :)